__tests__/app.test.js

```diff
@@ -66,4 +67,45 @@ describe('2. GET /api/topics', ()=>{
             expect(msg).toBe('Not Found');
         })
     });
+});
+
+describe('3. GET /api/articles', ()=>{
+    test('responds with an array of article objects', ()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body: {articles}})=>{
+            expect(articles).toBeInstanceOf(Array);
```

Comment:
Array check is often redundant when we're using a forEach lower down (it has to be an array for the loop to work)